src/server/db/mongo/user-privat-provider.js

```diff
@@ -98,6 +98,21 @@ class UserPrivate {
     return await this.model.findOne({ identifier }).lean()
   }
 
+  /**
+   * complete Step by identifier and step name
+   *
+   * @param {int} identifier
+   * @param {string} stepName
+   *
+   * @returns {object || null}
+   */
+  async completeStep(identifier, stepName) {
```

Comment:
this is not the correct way to update a field in mongodb

Reply:
made updateOne method